apps/dashboard/src/app/event/components/pools-box.tsx

```diff
@@ -1,38 +1,34 @@
 import { notifyFail } from "@/lib/notifications"
-import type { AttendancePool } from "@dotkomonline/types"
+import type { Attendance, AttendancePool } from "@dotkomonline/types"
 import { Box, Button, Card, Flex, Grid, Text, Title } from "@mantine/core"
 import type { FC } from "react"
 import { useDeletePoolMutation } from "../mutations"
 import { openEditPoolModal } from "./edit-pool-modal"
 
-interface PoolsBoxProps {
-  pools: AttendancePool[]
-  attendanceId: string
-}
-
 interface NormalPoolBoxProps {
   pool: AttendancePool
-  attendanceId: string
+  attendance: Attendance
   deleteGroup: (id: string, numAttendees: number) => void
 }
 
-const AttendancePoolCard: FC<NormalPoolBoxProps> = ({ pool, attendanceId, deleteGroup }) => {
+const AttendancePoolCard: FC<NormalPoolBoxProps> = ({ pool, attendance, deleteGroup }) => {
+  const attendeeCount = attendance.attendees.filter((attendee) => attendee.attendancePoolId === pool.id).length
   return (
     <Card shadow="sm" padding="lg" radius="md" withBorder key={pool.id} mt={16}>
       <Flex justify="space-between" direction="column" gap="lg">
         <Box>
           <Title order={2}>{pool.title}</Title>
           <Text>
             {pool.capacity > 0
-              ? `${Math.min(pool.numAttendees, pool.capacity)} / ${pool.capacity} påmeldte`
+              ? `${Math.min(attendeeCount, pool.capacity)} / ${pool.capacity} påmeldte`
               : "Ledige plasser"}
           </Text>
-          {pool.numAttendees - pool.capacity > 0 && <Text>{pool.numAttendees - pool.capacity} på venteliste</Text>}
+          {attendeeCount - pool.capacity > 0 && <Text>{attendeeCount - pool.capacity} på venteliste</Text>}
```

Comment:
And this should do a `.filter()` for unreserved attendees, since attendee count isn't necessarily ceiled by capacity

Reply:
Where should the filter be..?